src/scenes/Event/components/EventTxHistory/index.js

```diff
@@ -58,11 +51,13 @@ export default class EventTxHistory extends Component {
                 <TableCell padding="dense">
                   <FormattedMessage id="str.status" defaultMessage="Status" />
                 </TableCell>
+                <TableCell padding="dense">
+                </TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
-              {_.map(transactions, (transaction, index) => (
-                this.renderTxRow(transaction, index)
+              {transactions.map((transaction) => (
+                <TxRow key={transaction.txid} transaction={transaction} intl={intl} />
```

Comment:
1. we don't need to pass `intl` here if we're using `@injectIntl` on our `TxRow` component
2. let's spread `transaction` here like `{...transaction}`